proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/server/rest/DigitalObjectResource.java

```diff
@@ -867,6 +867,12 @@ public DublinCoreRecord updateDublinCore(DublinCoreRecord update) throws IOExcep
         dMetadata.setIgnoreValidation(ignoreValidation);
         try {
             if (isJsonData) {
+                //#458 type normalPage should not be stored
```

Comment:
Přesunuto do NDKMetadataHandleru. viz metoda fixPageType(...) a její použití v setMetadataAsJson(...).